packages/chat/src/Request/AIChatRequest.php

```diff
@@ -112,8 +116,18 @@ public function matches(array $criteria): bool
         return $this->criteria->matches($criteria);
     }
 
+    public function handleSchema(): void
+    {
+        $this->options['schemaHandled'] = true;
```

Comment:
@coderabbitai how would you handle that? have you a better idea?

Reply:
@coderabbitai the issue with that is that when the request is build the system does not know if the adapter (choosen later) will handle the schema (see the OpenAiChatAdapter) or it should be handled automatically.